gatsby-config.js

```diff
@@ -270,6 +270,7 @@ module.exports = {
   plugins,
   siteMetadata: {
     description,
+    author: 'Iterative',
```

Comment:
I'm not sure if this is actually used anywhere, I at least don't see it in the SEO component's query, though there's a case for having a single place to define this for our plans to make the docs engine more reusable.

Reply:
:shrug: 'twas just copied over from cml.dev